docs/modules/candid-guide/pages/candid-howto.adoc

```diff
@@ -56,19 +55,50 @@ For more information about using `+dfx+` and the `+dfx canister call+` command,
 The Candid interface description language provides a common language for specifying the signature of a canister service.
 Based on the type signature of that service, Candid provides a web interface—the Candid UI—that allows you to call canister functions for testing and debugging from a web browser without writing any front-end code.
 
-To use the Candid web interface to test `counter `canister:
+To use the Candid web interface to test the `counter `canister:
 
-. Find the canister identifier associated with the `counter` canister using the `dfx canister id counter` command.
+. Find the Candid UI canister identifier associated with the `counter` canister using the `+dfx canister id __Candid_UI+` command.
++
+[source,bash]
+----
+dfx canister id __Candid_UI
+----
++
+The command displays the canister identifier for the Candid UI with output similar to the following:
++
+....
+r7inp-6aaaa-aaaaa-aaabq-cai
+....
+. Start the {IC} locally by running the following command:
++
+[source,bash]
+----
+dfx start --background
```

Comment:
--background is still buggy and almost always fails for me after 5-10 min.

Reply:
buggy, meaning it doesn't stay in the background?

Reply:
correct.

Reply:
I don't think we need to mention `dfx start` here.  The whole tutorial assumes there is a replica running. Let me do a pass on this, and add instructions for accessing from the mainnet.

Reply:
I would like to have instructions for using the Candid UI both locally (`dfx start` required) and running on `--network ic` with the new `_` URL